run_pacta_data_preparation.R

```diff
@@ -890,6 +890,38 @@ for (pkg_name in pacta_packages) {
   )
 }
 
+# Create archive files
+if (config[["export_archives"]]) {
```

Comment:
```suggestion
if (export_archives) {
```
Generally speaking, I currently prefer this approach, but to remain in-sync with how other parameters are currently being used/managed in this script, I'd rather see a line like `export_archives <- config$export_archives` above, e.g. between [these two lines](https://github.com/RMI-PACTA/workflow.data.preparation/blob/e27b2d443a6322874d4883187275cd41bf6b30cd/run_pacta_data_preparation.R#L50C1-L51), and maybe in the future we convert everything to the strategy of using parameters directly from the `config` object/list.

Reply:
tracking switch to new method in #160